chain/types/ethtypes/eth_types.go

```diff
@@ -826,6 +826,44 @@ func GetContractEthAddressFromCode(sender EthAddress, salt [32]byte, initcode []
 	return ethAddr, nil
 }
 
+type FilecoinAddressToEthAddressParams struct {
+	FilecoinAddress address.Address
+	BlkParam        *string
```

Comment:
```suggestion
	FilecoinAddress address.Address `json:"address"`
	Block        *string `json:"block,omitempty"`
```

Are we following a pattern with `BlkParam`? seems odd to name it

this `json` tagging isn't really critical since this is internal, just nice to be consistent

Reply:
actually forget this, it's never touched